packages/loader/container-loader/src/loadPaused.ts

```diff
@@ -59,7 +60,7 @@ export async function loadContainerPaused(
 	// Force readonly mode - this will ensure we don't receive an error for the lack of join op
 	container.forceReadonly?.(true);
 
-	const dm = container.deltaManager;
+	const dm = container.deltaManager as IDeltaManagerInternal;
```

Comment:
Feels like we should have a type-guard function that guarantees something is an `IDeltaManagerInternal` instead of the cast.